### PR TITLE
Ansh sponsors fix + removed frontend files from original repo

### PR DESCRIPTION
**Pull request by naveennvrgup**

### CODE REVIEW

File: startup/models.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from django.db import models
+#from appprofile.models import Profile
+# Create your models here.
+class startup(models.Model):
```
Comment (kdpisda) [inline startup/models.py]: `Startup` not startup @naveennvrgup

File: startup/serializers.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import startup
+
+class startupSerializer(serializers.ModelSerializer):
```
Comment (kdpisda) [inline startup/serializers.py]: `StartupSerializer` @naveennvrgup

**Review by kdpisda — CHANGES_REQUESTED** [review]

File: startup/models.py (hunk 1 above)
Comment (naveennvrgup) [reply startup/models.py]: these are the commits of @anshsrtv. i hav just cleaned his old PR #33

File: startup/serializers.py (hunk 2 above)
Comment (naveennvrgup) [reply startup/serializers.py]: these are the commits of @anshsrtv. i hav just cleaned his old PR #33

File: startup/models.py (hunk 1 above)
Comment (naveennvrgup) [reply startup/models.py]: will fix it

Comment (naveennvrgup) [reply startup/models.py]: sir this is cleanup PR of PR #33 made by @anshsrtv. for some reason he have deleted these files on commit d8f9e95(if u check PR #33) for which u have been asking changes for. So these files dont exit in this branch. I just reset --soft to the commit he added the outdated frontend files and removed them and made a PR.

File: startup/serializers.py (hunk 2 above)
Comment (naveennvrgup) [reply startup/serializers.py]: sir this is cleanup PR of PR #33 made by @anshsrtv. for some reason he have deleted these files on commit d8f9e95(if u check PR #33) for which u have been asking changes for. So these files dont exit in this branch. I just reset --soft to the commit he added the outdated frontend files and removed them and made a PR.

**Pull request merged**